server/src/exceptions/RestaurantAlreadyInFavourites.ts

```diff
@@ -0,0 +1,10 @@
+import HttpException from './HttpException';
+
+class RestaurantAlreadyInFavourites extends HttpException {
+    constructor(id?: string) {
+      const idErrorMessage = `Restaurant with this id is already in favourites`;
+      super(400, idErrorMessage);
+    }
+}
+
+export default RestaurantAlreadyInFavourites;
```

Comment:
jak przyjmujesz zmienną id do funkcji, to jej użyj :)

Reply:
Zrobione ;)